engine/api/environment_import.go

```diff
@@ -71,13 +69,14 @@ func (api *API) postEnvironmentImportHandler() service.Handler {
 
 		_, msgList, globalError := environment.ParseAndImport(tx, api.Cache, proj, eenv, force, project.DecryptWithBuiltinKey, getUser(ctx))
 		msgListString := translate(r, msgList)
-
 		if globalError != nil {
-			myError, ok := globalError.(sdk.Error)
-			if ok {
-				return service.WriteJSON(w, msgListString, myError.Status)
+			globalError = sdk.WrapError(globalError, "Unable to import environment %s", eenv.Name)
+			if sdk.ErrorIs(globalError, sdk.ErrUnknownError) {
+				return globalError
```

Comment:
it's not exactly the same thing here...

Reply:
I created a sdk.ErrorIsUnknown and updated the ErrorIs to return true is an error not known (not from our sdk) is compared with ErrUnknownError.